### PR TITLE
refactor: merge tool and monitor process.

### PR DESCRIPTION
**Pull request by re2zero**
Merge tool and monitor process as one share library, and then lunch it as plugin from dde-file-manager-daemon. From this version, there is no deepin anything related service, request the same D-bus API if need.

Log: change the process to share library.

### CODE REVIEW

File: server/backend/anythingbackend.cpp [hunk 1]
```diff
@@ -176,11 +206,52 @@ int main(int argc, char *argv[])
...
+    logger->registerAppender(rollingFileAppender);
+
+    for (const QString &c : LFTManager::logCategoryList()) {
+        logger->registerCategoryAppender(c, consoleAppender);
+        logger->registerCategoryAppender(c, rollingFileAppender);
+    }
+
+     QDBusConnection connection = QDBusConnection::systemBus();
+     if (!connection.interface()->isServiceRegistered(anythingServicePath)) {
+         bool reg_result = connection.registerService(anythingServicePath);
```
[the remark this answers is not in the document]
Comment (re2zero) [reply server/backend/anythingbackend.cpp:239]: new push fixed.

**Comment by re2zero** [comment]
/+1
/review @wangrong1069

**Review by wangrong1069 — APPROVED** [review]
looks good for me.

**Pull request merged**